src/ol/View.js

```diff
@@ -448,6 +443,36 @@ class View extends BaseObject {
     this.options_ = options;
   }
 
+  /**
+   * Padding (in css pixels).
+   * If the map viewport is partially covered with other content (overlays) along
+   * its edges, this setting allows to shift the center of the viewport away from that
+   * content. The order of the values in the array is top, right, bottom, left.
+   * The default is no padding, which is equivalent to `[0, 0, 0, 0]`.
+   * @type {Array<number>|undefined}
+   * @api
+   */
+  get padding() {
+    return this.padding_;
+  }
+  set padding(padding) {
```

Comment:
Were people trying to change the padding by setting `view.padding`?

Not opposed to the setter, just noticing that it is different than the rest of the API (aside from MapBrowserEvent).

Reply:
I see that `padding` was marked as part of the API in 6.5.0 (20ccdc56b8e6f7d5fcf2642dccb79d2777950ac7).

Reply:
Yes, people were trying to change the padding by setting `view.padding` because I had marked it as part of the API. Back then I had not thought about the consequences thoroughly enough, so I had to use a setter now.